examples/v1beta1/trial-images/enas-cnn-cifar10/Dockerfile.cpu

```diff
@@ -1,12 +1,11 @@
-FROM tensorflow/tensorflow:1.15.4-py3
+FROM tensorflow/tensorflow:2.7.0
```

Comment:
Does it still work with 1.15? Or do we have to use 2.7 here?

Reply:
I have checked the following on my local machine about an example using TF 1.15.
Should I leave a separate example using TensorFlow 1.15?
WDTY? @gaocegege @andreyvelich @johnugeorge 

```shell
$ kubectl get cm katib-config -o jsonpath='{.data.suggestion}' | jq '.enas'
{
  "image": "ytenzen/suggestion-enas:experimental-amd64",
  "resources": {
    "limits": {
      "memory": "200Mi"
    }
  }
}
$ kubectl apply -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/nas/enas-cpu.yaml
$ kubectl describe experiments.kubeflow.org enas-cpu -n kubeflow
Name:         enas-cpu
Namespace:    kubeflow
Labels:       <none>
Annotations:  <none>
API Version:  kubeflow.org/v1beta1
Kind:         Experiment
Spec:
  Algorithm:
    Algorithm Name:        enas
  Trial Template:
    Trial Spec:
      API Version:  batch/v1
      Kind:         Job
      Spec:
        Template:
          Spec:
            Containers:
              Command:
                python3
                -u
                RunTrial.py
                --num_epochs=1
                --architecture="${trialParameters.neuralNetworkArchitecture}"
                --nn_config="${trialParameters.neuralNetworkConfig}"
              Image:         docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v1beta1-45c5727
              Name:          training-container
            Restart Policy:  Never
Status:
  Completion Time:  2021-11-15T09:42:19Z
  Conditions:
    Last Transition Time:  2021-11-15T09:38:35Z
    Last Update Time:      2021-11-15T09:38:35Z
    Message:               Experiment is created
    Reason:                ExperimentCreated
    Status:                True
    Type:                  Created
    Last Transition Time:  2021-11-15T09:42:19Z
    Last Update Time:      2021-11-15T09:42:19Z
    Message:               Experiment is running
    Reason:                ExperimentRunning
    Status:                False
    Type:                  Running
    Last Transition Time:  2021-11-15T09:42:19Z
    Last Update Time:      2021-11-15T09:42:19Z
    Message:               Experiment has succeeded because max trial count has reached
    Reason:                ExperimentMaxTrialsReached
    Status:                True
    Type:                  Succeeded
  Current Optimal Trial:
    Best Trial Name:  enas-cpu-hv8fqzkb
    Observation:
      Metrics:
        Latest:  0.20869999969005584
        Max:     0.20869999969005584
        Min:     0.20869999969005584
        Name:    Validation-Accuracy
    Parameter Assignments:
      Name:    architecture
      Value:   [[38]]
      Name:    nn_config
      Value:   {'num_layers': 1, 'input_sizes': [32, 32, 3], 'output_sizes': [10], 'embedding': {'38': {'opt_id': 38, 'opt_type': 'separable_convolution', 'opt_params': {'filter_size': '3', 'num_filter': '64', 'stride': '1', 'depth_multiplier': '1'}}}}
  Start Time:  2021-11-15T09:38:35Z
  Succeeded Trial List:
    enas-cpu-jwzrkhjr
    enas-cpu-mbddqsq2
    enas-cpu-hv8fqzkb
  Trials:            3
  Trials Succeeded:  3
Events:              <none>
```

Reply:
I have no objection to it. I just want to make sure that we can support 1.x TF.

Reply:
We will stop supporting TF v1.X as discussed in [this](https://github.com/kubeflow/katib/pull/1731#issuecomment-975688250).